cmd_list_test.go

```diff
@@ -1663,3 +1662,164 @@ func TestLmove(t *testing.T) {
 		)
 	})
 }
+
+func TestBlmove(t *testing.T) {
+	s, err := Run()
+	ok(t, err)
+	defer s.Close()
+	c, err := proto.Dial(s.Addr())
+	ok(t, err)
+	defer c.Close()
+
+	t.Run("Behaves just like LMOVE", func(t *testing.T) {
+		s.Push("src", "LR", "LL", "RR", "RL")
+		s.Push("dst", "m1", "m2", "m3")
+
+		// RIGHT LEFT
+		{
+			mustDo(t, c,
+				"LMOVE", "src", "dst", "RIGHT", "LEFT",
```

Comment:
I guess you need to use `BLMOVE` here, not `LMOVE`. Same for the other tests, and the ones in integration.

Reply:
Ops, big oversight on my part. I made the changes and it helped me find a bug: my blockCmd was returning false in a couple of places where I was supposed to return true. :tada: